### PR TITLE
{AzureCXP} fixes Azure/azure-powershell#18319

### PR DESCRIPTION
**Pull request by navba-MSFT**
Examples New-AzRecoveryServicesAsrProtectionContainerMapping and Get-AzRecoveryServicesAsrReplicationProtectedItem use the variable $PrimaryContainer, but don't tell you how that variable is being set.

This PR mentions address this issue. Adds the below point to clear the confusion:

To get the ASR protection container in the Recovery Service Container run the [Get-AzRecoveryServicesAsrProtectionContainer](https://docs.microsoft.com/en-us/powershell/module/az.recoveryservices/get-azrecoveryservicesasrprotectioncontainer?view=azps-8.0.0) PS command.

Fixes Azure/azure-powershell#18319

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesAsrReplicationProtectedItem.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ Accept wildcard characters: False
 ```
 
 ### -ProtectionContainer
-Specifies the ASR protection container object of the ASR protection container corresponding to the replication protected item.
+Specifies the ASR protection container object of the ASR protection container corresponding to the replication protected item. To get the ASR protection container in the Recovery Service Container run the [Get-AzRecoveryServicesAsrProtectionContainer](https://docs.microsoft.com/en-us/powershell/module/az.recoveryservices/get-azrecoveryservicesasrprotectioncontainer?view=azps-8.0.0) PS command.
```
Comment (mikefrobbins) [inline src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesAsrReplicationProtectedItem.md:124]:
```suggestion
Specifies the ASR protection container object of the ASR protection container corresponding to the replication protected item. To get the ASR protection container in the Recovery Service Container run the [Get-AzRecoveryServicesAsrProtectionContainer](/powershell/module/az.recoveryservices/get-azrecoveryservicesasrprotectioncontainer) cmdlet.
```

File: src/RecoveryServices/RecoveryServices/help/New-AzRecoveryServicesAsrProtectionContainerMapping.md [hunk 2]
```diff
@@ -132,7 +132,7 @@ Accept wildcard characters: False
 ```
 
 ### -PrimaryProtectionContainer
-Specifies the ASR protection container object for the  primary protection container to be used in the mapping.
+Specifies the ASR protection container object for the  primary protection container to be used in the mapping. To get the ASR protection container in the Recovery Service Container run the [Get-AzRecoveryServicesAsrProtectionContainer](https://docs.microsoft.com/en-us/powershell/module/az.recoveryservices/get-azrecoveryservicesasrprotectioncontainer?view=azps-8.0.0) PS command.
```
Comment (mikefrobbins) [inline src/RecoveryServices/RecoveryServices/help/New-AzRecoveryServicesAsrProtectionContainerMapping.md:135]:
```suggestion
Specifies the ASR protection container object for the primary protection container to be used in the mapping. To get the ASR protection container in the Recovery Service Container run the [Get-AzRecoveryServicesAsrProtectionContainer](/powershell/module/az.recoveryservices/get-azrecoveryservicesasrprotectioncontainer) cmdlet.
```

**Review by mikefrobbins — CHANGES_REQUESTED** [review]
URL's should be relative so they work in air-gapped environments. They also shouldn't include specific culture or version information. I would also recommend avoiding the use of abbreviations such as _PS_ for PowerShell.

**Comment by VeryEarly** [comment]
please go to these cmdlets, update parameter descriptions there, and re-generate help markdown.
Otherwise your doc fix will be overwritten next time docs are generated

**Comment by navba-MSFT** [comment]
@mikefrobbins Your suggested changes have been committed. Could you review and approve this PR ?

**Review by mikefrobbins — DISMISSED** [review]
LGTM

**Comment by navba-MSFT** [comment]
@mikefrobbins @VeryEarly Could you please merge this PR once you get a chance ?

**Comment by VeryEarly** [comment]
> @mikefrobbins @VeryEarly Could you please merge this PR once you get a chance ?

Could you please resolve my comments?

**Comment by navba-MSFT** [comment]
> > @mikefrobbins @VeryEarly Could you please merge this PR once you get a chance ?
> 
> Could you please resolve my comments?

@VeryEarly I just did. I updated the parameter descriptions in the cmdlets. Could you please Review and approve once you get a chance ?

**Review by VeryEarly — DISMISSED** [review]

**Review by mikefrobbins — DISMISSED** [review]
LGTM

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**